src/Microsoft.DotNet.XHarness.Android/Execution/AdbProcessManager.cs

```diff
@@ -67,27 +67,33 @@ public ProcessExecutionResults Run(string adbExePath, string arguments, TimeSpan
             p.BeginOutputReadLine();
             p.BeginErrorReadLine();
 
-            // Sleeping 1 second allows the process time to send messages to the above delegates
-            // if the process exits very quickly
-            Thread.Sleep(1000);
-
             bool timedOut = false;
+            int exitCode = -1;
```

Comment:
maybe make this a large weird # to make it clear it happened.

Reply:
Hm yeah good point, I think ADB even returns -1 for some "normal" case. Will fix.